pkgs/development/perl-modules/Bio-BigFile/default.nix

```diff
@@ -0,0 +1,66 @@
+{
+ BioPerl,
+ IOString,
+ buildPerlModule,
+ fetchFromGitHub,
+ fetchpatch,
+ fetchurl,
+ kent,
+ lib,
+ libmysqlclient,
+ libpng,
+ openssl,
+ perl
+}:
+
+buildPerlModule rec {
+  pname = "Bio-BigFile";
+  version = "1.07";
+
+  src = fetchurl {
+    url = "mirror://cpan/authors/id/L/LD/LDS/Bio-BigFile-${version}.tar.gz";
+    sha256 = "277b66ce8acbdd52399e2c5a0cf4e3bd5c74c12b94877cd383d0c4c97740d16d";
+  };
+
+  # Only kent 335 works with Bio-BigFile, see
+  # - official documentation: https://www.ensembl.org/info/docs/tools/vep/script/vep_download.html#bigfile
+  # - one of the developer's answer: https://github.com/Ensembl/ensembl-vep/issues/1412
+  # KENT_SRC is needed for building BioBigfil
+  KENT_SRC = kent.overrideAttrs (old: rec {
```

Comment:
The attribute name `KENT_SRC` is not good in my opinion - `kent'` is more common for this case, the `_SRC` suffix is incorrect - this is not a source derivation.

Reply:
BioBigfile needs an environment variable named KENT_SRC to know where kent is installed. I cannot change that unfortunately.